packages/front-end/components/SchemaBrowser/EditSqlModal.tsx

```diff
@@ -86,6 +190,9 @@ export default function EditSqlModal({
       open
       header="Edit SQL"
       submit={form.handleSubmit(async (value) => {
+        if (queryType === "experiment-assignment") {
+          await handleTestQuery();
```

Comment:
There are some valid use cases for saving a query which throws errors.  For example, in Segment, tables don't exist until you track at least 1 event.  So if someone has never done experimentation before, they might connect their data source before integrating our SDK and tracking the `viewed experiment` event.  In that case, the query would fail to run today, but would work in the future.

Can we make this a non-blocking operation so they are still allowed to save if the query fails (maybe with a warning)?